ci/scripts/helpers.js

```diff
@@ -100,4 +100,9 @@ const comment = ({ prNumber, body }) => {
     exec('gh', ['pr', 'comment', `${prNumber}`, '--body', body]);
 };
 
-module.exports = { checkPackageDependencies, exec, commit, comment };
+const triggerWorkflowSync = async (branchName, workflows) => {
```

Comment:
thats an interesting approach. I only have one problem with it - this way we coulde easily end up in situation where a single workflow is triggered multiple times. once by incorrect `on` rules in yaml files and once by this script 🤔 

on the other hand I don't have a better idea how to do it.

Reply:
@mroz22 I created new PR https://github.com/trezor/trezor-suite/pull/11877  where I am changing the path of the release to ` release/**` instead of `release/* `, I think that is the reason why the actions did not run on previous release. And that makes all this `triggerWorkflowSync` unnecessary, So I can close this PR.